server/sonar-server/src/test/java/org/sonar/server/qualitygate/RegisterQualityGatesTest.java

```diff
@@ -52,32 +63,165 @@
 
   @Rule
   public DbTester db = DbTester.create(System2.INSTANCE);
+  @Rule
+  public LogTester logTester = new LogTester();
+
+  private static final String A_RATING_VALUE = Integer.toString(RatingGrid.Rating.A.getIndex());
+  private static final int LEAK_PERIOD = 1;
+  private static final String BUILT_IN_NAME = "Sonar way";
+
+  private DbClient dbClient = db.getDbClient();
+  private DbSession dbSession = db.getSession();
+
+  private QualityGateDao qualityGateDao = dbClient.qualityGateDao();
```

Comment:
same as prod code, on my side I prefer not having too much dao variables, instead I directly use dbClient.XXXdao()

Reply:
Can do this, on my side I prefer this way like *dbClient* or *dbSession* that make the code more clearer with long method names.

Reply:
ok